news/2018-05-26-beatmap-spotlights-april-2018.md

```diff
@@ -0,0 +1,144 @@
+
+The CHALLENGE difficulty is where the fun begins! The vocal tells you how to play and then the pattern repeat again but with a incredibly high slider velocity, high enough that the notes become unreadable. You have to remember which pattern you played before and listen to the vocal to hit them accurate!
+
+This mapset really deserve it to be the best of the month April. Well done Lenfried-! Would be cool to see more maps like this from you!
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/deublithick.jpg)](http://osu.ppy.sh/beatmapsets/725078)
+
+[\_underjoy](https://osu.ppy.sh/users/2235750) came back with his undoubtfully insane 7K mapset with Gekikara Mania's "Deublithick". The spread features with three extreme difficulties, including [Blocko](https://osu.ppy.sh/users/4075092) as a guest mapper and an Insane difficulty accompanied by [capsafangirl](https://osu.ppy.sh/users/4308015), which present extreme streaming burst at the 1/3 posterior section of the song, along with complex usage of multiple chord stream jump hands.
+
+\_underjoy once again showed his unique and excellent mapping style, encouraging for 7K lovers to once again challenge and test their skills.
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/onigiri.jpg)](http://osu.ppy.sh/beatmapsets/663098)
+
+Debuting his first ranked map, [Mentholzzz](https://osu.ppy.sh/users/7182779)'s ONIGIRI FREEWAY by featured artist OISHII is quite the active chart for an active song. Featuring twisting long notes with well crafted inverse sections, it's certainly a break from the established norm. 
+
+Even if long notes aren't your thing, the lower difficulties share the same creative spark that makes the top diff stand out, and to top it all off, it's key sounded! With a little something for everyone, this has definitely earned its place in this month's Spotlights.
+
+[![](/wiki/shared/news/2018-05-26-beatmap-spotlights-april-2018/yell.jpg)](http://osu.ppy.sh/beatmapsets/661022)
+
+[Turrim](https://osu.ppy.sh/users/3846265), new mania mapper has made his appearance with his first ranked mapset, DJ Genki VS Camellia feat. moimoi - YELL!
+
+As he showed on many unranked maps, this map also has kind of unusual long note style. There is the well-layered structure, starting from vocal to heavy drum roll backs it up. He focused on the feeling when he faced the part, it makes him to put many long notes at the main melody. Sometimes he excluded the strength of drum for the sake of it, that aids in organization of pattern creativity in the good way. 
+
+The last kiai is made up of full long notes, it gives a challenge to players who are not familiar with them. It's a new challenge for him as 7K mapping, let's give him a round of applause for his best effort!
```

Comment:
it just seems kinda silly to give a round of applause for effort in the spotlight lol, A for effort feels like he tried his best but didn't do well